be/src/storage/hll.h

```diff
@@ -238,8 +238,6 @@ class HllSetResolver {
 public:
     HllSetResolver()
 
-    {}
-
```

Comment:
You can't delete `{}`, compile is failed

Reply:
done